src/plugin/init/utils/qa/prompt.ts

```diff
@@ -81,9 +98,38 @@ export const promptForHomepage = async (
   });
 };
 
-export const promptForSupportMobile = async (m: BoundMessage) => {
-  return confirm({
-    default: true,
-    message: m("Q_MobileSupport"),
-  });
+export type LangAnswers = {
+  name: string;
+  description: string;
+  homepage?: string;
+};
+
+export const promptForLang = async <R extends boolean = false>(
+  m: BoundMessage,
+  options: {
+    required?: R;
+    supportLang: SupportLang;
+    defaultName: string;
+    defaultDescription?: string;
+  },
+): Promise<R extends true ? LangAnswers : Partial<LangAnswers>> => {
+  // NOTE: lang en is always required
+  if (!options.required && options.supportLang !== "En") {
```

Comment:
IMO: 追加ですみません。requiredプロパティを外から渡しているので、「英語だったら必須」という情報はこの関数内で持たなくて良さそうです。



```suggestion
  if (!options.required) {
```

Reply:
あーそこちょっと悩んだんですけど、下記箇所の文言が"En"だと型エラーになるのでここで弾いてます
(サポートするかどうかの質問文のkeyがここの型に依存している)
https://github.com/kintone/cli-kintone/blob/27e1183fd67ae31d4fe6ab69c595469191a94ec9/src/plugin/init/utils/qa/prompt.ts#L88C5-L88C43

Reply:
あーーなるほど！

- サポートするかどうかの文言を英語用も作る（promptForLangの汎用性を維持できるが、実際には使われない文言を用意することになる）
- promptForLang と　promptForOptionalLangを分ける（前者は実質英語専用になる）

など考えましたがどちらも微妙なのでこのままで良さそうです！